### ISSUE TITLE
Unable to download from Crunchyroll due to Request blocked by Cloudflare

### ISSUE THREAD

**Issue by YewTreeWeb** [yt-dlp/yt-dlp#8447]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.10.13** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

UK

### Provide a description that is worded well enough to be understood

I was able to download one episode from Crunchyroll using the cookies from the browser flag but after the first video no matter what I seem to try I cannot download any others or the video that previously worked. I have even tried downloading the cookie file manually with the recommended browser extension but that gives me the same error. The error I keep getting is `ERROR: [crunchyroll] GZ7UVX853: Request blocked by Cloudflare; navigate to Crunchyroll in your browser, then pass the fresh cookies (with --cookies-from-browser or --cookies) and your browser's User-Agent (with --user-agent)`. I have re-logged in with Chrome and re-run the command but still get this error. It's just odd that I downloaded one video but now can't. 

I have a static IP with my ISP provider and I use Cloudflares DNS settings. I have also cleared the data from the browser and tried again with no luck.

Any help you can give would be really appreciated

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp --cookies-from-browser chrome --cookies "cookies.txt" https://www.crunchyroll.com/watch/GZ7UVX853/someone-who-lives-in-the-exact-opposite-world-as-me --user-agent 'Mozilla/5.0' --verbose
[debug] Command-line config: ['--cookies-from-browser', 'chrome', '--cookies', 'cookies.txt', 'https://www.crunchyroll.com/watch/GZ7UVX853/someone-who-lives-in-the-exact-opposite-world-as-me', '--user-agent', 'Mozilla/5.0', '--verbose']
Extracting cookies from chrome
[debug] Extracting cookies from: "/Users/mat/Library/Application Support/Google/Chrome/Default/Cookies"
[debug] using find-generic-password to obtain password from OSX keychain
Extracted 48 cookies from chrome
[debug] cookie version breakdown: {'v10': 48, 'other': 0, 'unencrypted': 0}
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.10.13 [b634ba742] (pip)
[debug] Python 3.12.0 (CPython x86_64 64bit) - macOS-13.0.1-x86_64-i386-64bit (OpenSSL 3.1.3 19 Sep 2023)
[debug] exe versions: rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.19.0, brotli-1.1.0, mutagen-1.47.0, sqlite3-3.43.2, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1890 extractors
[crunchyroll] Extracting URL: https://www.crunchyroll.com/watch/GZ7UVX853/someone-who-lives-in-the-exact-opposite-world-as-me
[debug] [crunchyroll] Using cxApiParam=noaihdevm_6iyg0a8l0q
[crunchyroll] Authenticating with grant_type=etp_rt_cookie
ERROR: [crunchyroll] GZ7UVX853: Request blocked by Cloudflare; navigate to Crunchyroll in your browser, then pass the fresh cookies (with --cookies-from-browser or --cookies) and your browser's User-Agent (with --user-agent)
  File "/usr/local/Cellar/yt-dlp/2023.10.13/libexec/lib/python3.12/site-packages/yt_dlp/extractor/common.py", line 715, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.10.13/libexec/lib/python3.12/site-packages/yt_dlp/extractor/crunchyroll.py", line 333, in _real_extract
    response = traverse_obj(self._call_api(
                            ^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.10.13/libexec/lib/python3.12/site-packages/yt_dlp/extractor/crunchyroll.py", line 127, in _call_api
    result = self._call_base_api(
             ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.10.13/libexec/lib/python3.12/site-packages/yt_dlp/extractor/crunchyroll.py", line 108, in _call_base_api
    self._update_auth()
  File "/usr/local/Cellar/yt-dlp/2023.10.13/libexec/lib/python3.12/site-packages/yt_dlp/extractor/crunchyroll.py", line 94, in _update_auth
    raise ExtractorError(
```

**Comment by Grub4K** [comment]
Please actually do whats suggested in the workaround. Your user agent is almost certainly not what you are passing

Besides, duplicate of https://github.com/yt-dlp/yt-dlp/issues/7442

**Comment by San3-Cod3** [comment]
My experience using a non-premium and free video - not sure how to get around this, have looked through the many different issues, but possibly older workarounds are outdated now? So looking at them further wouldn't help, and there are thousands of issues to scour - this seems close my issue already, so deciding to post this here.  I wonder how I could pass it my L3 CDM keys?

```
C:\Users\...\yt-dlp>yt-dlp --cookies-from-browser firefox --username "redacted" --password "redacted" --user-agent "Mozilla/5.0 (Wind
ows NT 6.1; Win64; x64; rv:115.14.0) Gecko/20100101 Firefox/115.14.0" -S vcodec:
h264,res:1080 --merge-output-format mkv --remux-video mkv https://www.crunchyrol
l.com/watch/GX9UQXJJ2/morning-light-falls-on-you
Extracting cookies from firefox
Extracted 1274 cookies from firefox
[crunchyroll] Extracting URL: https://www.crunchyroll.com/watch/GX9UQXJJ2/mornin
g-light-falls-on-you
[crunchyroll] Authenticating with grant_type=etp_rt_cookie
ERROR: [crunchyroll] GX9UQXJJ2: Request blocked by Cloudflare; navigate to Crunc
hyroll in your browser, then pass the fresh cookies (with --cookies-from-browser
 or --cookies) and your browser's User-Agent (with --user-agent)
 ```
